openstack/db/v1/instances/requests.go

```diff
@@ -47,6 +47,8 @@ func (opts NetworkOpts) ToMap() (map[string]interface{}, error) {
 
 // CreateOpts is the struct responsible for configuring a new database instance.
 type CreateOpts struct {
+	// The availability zone of the instance.
+	AvailabilityZone string `json:"availability_zone,omitempty"`
 	// Either the integer UUID (in string form) of the flavor, or its URI
 	// reference as specified in the response from the List() call. Required.
 	FlavorRef string
```

Comment:
weird that other fields are not json omitempty...

Reply:
@EmilienM 

Unforunately other fields doesn't have json annotations.

I added `omitempty`, because availability_zone is optional as I can see it in the Openstack API Docs.
What the option should I stay, with or without  `omitempty`?

Reply:
I'm fine how you did. Of course consistency would be nice at some point.

Reply:
Nice.